erigon-lib/state/domain.go

```diff
@@ -1574,146 +1482,6 @@ func (dt *DomainRoTx) GetLatestFromFiles(key []byte) (v []byte, found bool, file
 	return dt.getFromFiles(key)
 }
 
-func (dt *DomainRoTx) IteratePrefix(roTx kv.Tx, prefix []byte, it func(k []byte, v []byte) error) error {
```

Comment:
we will use tx.Range queries instead?

Reply:
we wont use anything, this is useless now